DarkFlippers/unleashed-firmware
# SubGhz some improvements

# What's new

 SubGhz app: removed duplicate code in `transmitter.c`
 SubGhz app: fix "add manual" Alutech AT 4N
 SubGhzRemote app: drop include from SubGhz app

 SubGhz: Improved code readability and some "if" optimization in custom button and keeloq
 Fix strange bug with formatting in VS CODE in `keeloq.c` and `star_line.c`

# Verification 

- check the functionality of custom buttons
- check "add manual" for Alutech AT 4N protocol in SubGhz app
- you may need to make changes to the added comments for better readability.

# Checklist (For Reviewer)
